R/runMDS.R

```diff
@@ -11,23 +11,26 @@
 #' @param ... For the \code{calculateMDS} generic, additional arguments to pass to specific methods.
 #' For the SummarizedExperiment and SingleCellExperiment methods, additional arguments to pass to the ANY method.
 #'
-#' For \code{runMDS}, additional arguments to pass to \code{calculateMDS}. 
-#' @param method String specifying the type of distance to be computed between cells.
+#' For \code{runMDS}, additional arguments to pass to \code{calculateMDS}.
+#' @param FUN a function returning a \code{dist} object from a matrix, where rows are samples and rows are features
```

Comment:
Capital `A`, end with a period.

Reply:
Also, it seems that `FUN` just needs to return a distance matrix, not necessary a `dist` object.

I don't really understand what you mean by "rows are samples and rows are features". You should say something like:

> A function that accepts a numeric matrix as its first argument, where rows are samples and columns are features; and returns a numeric distance matrix (or an object coercible into one) of order equal to the number of samples.